transform/macros/fact_event_type.sql

```diff
@@ -0,0 +1,5 @@
+{% macro fact_event_type(event_type) -%}
+  SELECT event_date, repo_id, user
+  FROM {{ ref("stg_gharchive") }}
+  WHERE event_type = "{{ event_type }}"
+{%- endmacro %}
```

Comment:
Shouldn't this PR include using this macro in the `fact_stars` and `fact_commits` tables?